Makefile

```diff
@@ -12,26 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+EMPTY :=
+SPACE := $(EMPTY) $(EMPTY)
+COMMA := $(EMPTY),$(EMPTY)
 
+BINARY = sonobuoy
 TARGET = sonobuoy
 GOTARGET = github.com/heptio/$(TARGET)
 REGISTRY ?= gcr.io/heptio-images
 IMAGE = $(REGISTRY)/$(TARGET)
 DIR := ${CURDIR}
 DOCKER ?= docker
+LINUX_ARCH := amd64 arm64
+DOCKERFILE :=
+PLATFORMS := $(subst $(SPACE),$(COMMA),$(foreach arch,$(LINUX_ARCH),linux/$(arch)))
 
 GIT_VERSION ?= $(shell git describe --always --dirty)
 IMAGE_VERSION ?= $(shell git describe --always --dirty)
 IMAGE_BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD | sed 's/\///g')
 GIT_REF = $(shell git rev-parse --short=8 --verify HEAD)
 
+GOBIN = /go/bin
```

Comment:
I don't want to need to be root to run this script. `GOBIN ?= /go/bin` maybe? Are you building this on a machine that doesn't have go set up?

Reply:
hi Liztio,
I have updated it, please check it.
Thanks.